packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -101,23 +101,20 @@ class RenderAndroidView extends RenderBox with _PlatformViewGestureMixin {
   bool _isDisposed = false;
 
   /// The Android view controller for the Android view associated with this render object.
-  AndroidViewController get viewController => _viewController;
-  AndroidViewController _viewController;
+  AndroidViewController get viewController => controller as AndroidViewController;
+
   /// Sets a new Android view controller.
   ///
   /// `viewController` must not be null.
   set viewController(AndroidViewController viewController) {
```

Comment:
Prob better name this paramter so it doesn't conflict with the getter `viewController`

Reply:
I removed `viewController` and overrode `controller` instead.